src/common/sql.cpp

```diff
@@ -46,6 +46,7 @@ void AsyncThreadBody(const char* user, const char* passwd, const char* host, uin
     while (asyncRunning)
     {
         con.HandleAsync();
+        std::this_thread::sleep_for(200ms); // TODO: This is bad and ugly. Replace with something better.
```

Comment:
@zach2good how about [std::this_thread::yield()](https://en.cppreference.com/w/cpp/thread/yield)? as I understand it, that essentially tells the scheduler that we're done and other stuff can use the CPU so it's not busy looping.

Reply:
Hmm. Reading more, yield() still causes a high CPU load to be displayed even if other process and threads can still use the CPU time that would normally be used by a busy loop without yield().

Reply:
Yeah, I'm not familiar with yield, but just looks like an os-bound tiny sleep that we can't control the length of. I'd say the answer here is to eventually use a condition variable to signal when there is some more work to do, but a sleep is fine for now